### ISSUE TITLE
Display dropdown menu and links to give users access to account management features

### ISSUE THREAD

**Issue by sandysage** [mozilla-lockbox/lockbox-extension#442]
Implement a dropdown menu giving the user the ability to:

* review and update their FxA account
* sign out of Lockbox
* access options/preferences for Lockbox

<img width="249" alt="image" src="https://user-images.githubusercontent.com/757401/34845624-b98a3bbe-f6d2-11e7-9bd6-470e7afa2e85.png">

## Acceptance criteria
- [x] Should have a way to navigate it on a keyboard

## To do:
- [ ] needs discussion for keyboard accessibility to manage any changes to be made in its own story
- [ ] display dropdown menu when clicking on "user info" on full-screen
- [ ] open https://accounts.firefox.com/settings when clicking "Account"
- [ ] open Options when clicking "Preferences" menu item
- [ ] sign out when clicking "Sign out" menu item (blocked by #149)

**Comment by sandysage** [comment]
- [ ] needs discussion for keyboard accessibility to manage any changes to be made in its own story
